crates/cadenza-eval/src/eval.rs

```diff
@@ -1361,6 +1538,23 @@ mod tests {
         assert_eq!(eval_single("1 != 2").unwrap(), Value::Bool(true));
     }
 
+    #[test]
+    fn eval_operator_as_variable() {
```

Comment:
This should be a snapshot test in `test-data`. I'm actually surprised this even parses TBH